client/js/components/statement/assessmentTable/DpAssessmentTableCard.vue

```diff
@@ -558,11 +558,7 @@
                     With tiptap we can set obscure as prop always when the obscure button should be visible in the field,
                     because the permission check (featureObscureText) takes place in tiptap
                     -->
-                <dp-loading
-                  v-if="reloadStatementEditor"
-                  class="u-pb-0_5 u-pr-0_5 u-pt-0_25 u-1-of-2 border--right" />
```

Comment:
why did You remove the loading state?

Reply:
as pointed out in the PR description, the loading state was introduced in https://github.com/demos-europe/demosplan-core/pull/2521 to overcome reactivity issues but those were no real issues as the only bug was the absent of the "shortened" attribute in the api response.